### ISSUE TITLE
instruction identification doesn't work well for complex sentences

### ISSUE THREAD

**Issue by Palashio** [Palashio/libra#15]
sentences with a comma don't work well; i've assumed that users will only do things like "model x" or "please estimate y". When you create complex sentences it fails because there are many different varying part of speech locations

**Comment by Sar1hak** [comment]
Instead of sentence input, we can take input as "operation : variable".
example,
"model x" will become "model : x"
"please estimate y" will become "predict : y"
"calculate accuracy for training data" will become "accuracy : training_set"

**Comment by Palashio** [comment]
are you saying that they'd enter english statement: "Model median house value and calculate training data" --> we create the predict: median_house_value, accuracy: training set

OR 

they, themselves will send in the "predict : y" sort of dictionary in themselves.

**Comment by Sar1hak** [comment]
I was talking about the latter.
I know you want to make the task much simpler for the user but at the cost of space, meaning that bigger the process bigger will be the size of the library file the user will be downloading, these may even increase the time complexity.

**Comment by Palashio** [comment]
so right now all we're doing from the instruction is identifying the target. the user can't include any other specifications. 

here's what i'm thinking based off of what you said: what if we just had them separate specifications with commas/periods. So like let's say instead of just "Model median house value" they type in "Model median house value, Use training accuracy." This way it's a bit easier for us to process, and it doesn't take up too much space. 

This way, if they don't want any specifications they can also just enter in "Model median house value."
